progetto/ServerProject/src/server/AccountListMonitor.java

```diff
@@ -202,4 +204,66 @@ public int getMAXACCOUNTNUMBER()
         return this.MAXACCOUNTNUMBER;
     }
 
+
```

Comment:
LIENA 16:(scrivo qua perche nella linea 16 non posso)
private int MAXACCOUNTNUMBER; si puà mettere final baste che sia inizializato nel costruttore

Reply:
LINEA 19:

 private int length=0; se si riferisce all'arrey Account[] accountList; gli aray hanno già una proprietà length 
[https://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html](url)

> Finally, you can use the built-in length property to determine the size of any array. The following code prints the array's size to standard output:

 System.out.println(anArray.length);

> 

Se lo usi per motivi di concorrenza allora ignora pure quello che ho detto

Reply:
LINEA 47:
if(this.getNumberOfAccount()>=MAXACCOUNTNUMBER){

mi sembra un po' eccesivo usare un metodo all'interno della classe quando lo puoi dedurre da length o accountList.length